NuGet.config

```diff
@@ -3,5 +3,9 @@
   <packageSources>
     <add key="AspNetVNext" value="https://www.myget.org/F/aspnetcidev/api/v3/index.json" />
     <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
+    <add key="dotnet-cli" value="https://www.myget.org/F/dotnet-cli/api/v3/index.json" />
```

Comment:
We plan to mirror all the CLI, xunit and nuget packages from those feed to ours. Please don't checkin this before because it brings CoreCLR packages that bypass our "CLR gate" and things will break. @pranavkm is working on that right now